tests/test_query.py

```diff
@@ -12,8 +12,9 @@ def test_data():
     """Create toy data for two datasets for testing."""
```

Comment:
Just an FYI, I've created an issue #126 for testing outcomes when `RETURN_AGG` is true vs false, following our observation today that such a test currently doesn't exist

Reply:
Thanks!